src/test/java/.gitignore

```diff
@@ -0,0 +1,19 @@
+# Package Files #
```

Comment:
I think Faisal has updated .gitignore at the root of the project with Fortify files that need to be ignored. Not sure we need this new .gitignore?

Reply:
I think I missed that one when I was merging from master